canonical-websites/www.ubuntu.com
# Add maas ebook to /download/server/thank-you page

## Done

* Add maas ebook to [/download/server/thank-you](https://docs.google.com/document/d/1S5zY7HN_Q8eQWch0mCEfLICfj8pW1bct9_wGArCNIqc/edit#) page by making the one on /download/server/provisioning an include
* Remove ‘Installing Ubuntu Server’ from download server subnav as it just redirects to tutorials.u.c now

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [thank-you](http://0.0.0.0:8001/download/server/thank-you) and [provisioning](http://0.0.0.0:8001/download/server/provisioning)
- see that the form looks and works correctly
